Source/WebCore/dom/BoundaryPoint.h

```diff
@@ -35,8 +35,8 @@ struct BoundaryPoint {
 
     BoundaryPoint(Ref<Node>&&, unsigned);
 
-    Document& document() const;
-    WEBCORE_EXPORT Ref<Document> protectedDocument() const;
+    inline Document& document() const;
```

Comment:
Personally, I like adding a comment stating *where* the function is defined.

Reply:
Okay. I felt a bit redundant when functions are marked inline and defined in the corresponding `Inlines.h` file but we can add a comment.